### DISCUSSION TITLE
Is there API to indicate a stream is ready to play? Have to press button twice to play

### DISCUSSION THREAD

**Discussion by martiankoder** [OvenMediaLabs/OvenMediaEngine#1178] [Help]
Hi,
 I am testing out OvenPlayer and OME. But I have to hit the play button twice, first one is to load the stream. The second one is to play.  I want to hit the button only one time and it plays.  I does not make sense to hit it twice as a user. Is there a  OME API to get a stream status or some thing to indicate the video is ready to play , so then I can call overplayer.play() or something like this? Thank you.

**Comment by getroot** [comment]
It's hard to answer as I don't know exactly what you want. If the stream exists on the server, click the play button once (this is a browser limitation) and it will play. What does clicking to "Load" mean? Are you talking about the Load button of https://demo.ovenplayer.com?
A more detailed description of your situation will help us to respond.

https://github.com/AirenSoft/OvenPlayer/issues/339 I'm sure this issue has already been answered by you, but here's the server's stream list lookup API:

https://airensoft.gitbook.io/ovenmediaengine/rest-api/v1/virtualhost/application/stream#get-stream-list
